frontend/src/App.js

```diff
@@ -26,62 +28,54 @@ const imgStyle = {
 };
 
 let gridItem = {
-  margin : "60px auto",
+  margin : "50px auto",
   float: "none",
   borderStyle: "solid",
   borderColor: "#e6e6e6",
   borderWidth: 1,
   borderRadius: 3,
   padding: "0",
   background: "#fff",
-  
+
 };
 let gridText = {
   margin: 10,
 }
 
-let userPosts = [{author: "nan",
-url: "https://www.gettyimages.fi/gi-resources/images/CreativeLandingPage/HP_Sept_24_2018/CR3_GettyImages-159018836.jpg",
-title:"My First Post",
-createdAt:"Today",
-description:"REEEEEEEEEEEEEEEEEEEEEEEEEEE SPAM SPAM SPAM SPAM SPAM SPAM SPAM SPAM margin: 10 margin: 10margin: 10margin: 10margin: 10margin: 10margin: 10margin: 10margin: 10margin: 10"},
-{author: "Wibly2",
-url: "https://www.gettyimages.fi/gi-resources/images/CreativeLandingPage/HP_Sept_24_2018/CR3_GettyImages-159018836.jpg",
-title:"My second Post",
-createdAt:"Yesterday",
-description:"This was my second post"},
-{author: "kariez",
-url: "https://www.gettyimages.fi/gi-resources/images/CreativeLandingPage/HP_Sept_24_2018/CR3_GettyImages-159018836.jpg",
-title:"My worst Post",
-createdAt:"today",
-description:"Dont"}];
-
 class App extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      posts: userPosts
+      posts: {}
     };
 
+    this.RefreshSite();
+
     this.updateInfo = this.updateInfo.bind(this);
   }
   updateInfo() {
     let u = document.querySelector("#imgUrl");
     let t = document.querySelector("#title");
     let d = document.querySelector("#desc");
-    userPosts.push({
-      author: "kariez",
-      url: u.value,
-      title: t.value,
-      createdAt: "today",
-      description: d.value
-    });
-
-    this.setState({
-      posts: userPosts
-    });
+
+    axios.post("http://localhost:3002/posts/1", qs.stringify({image: u.value, title: t.value, description: d.value, username: "BestUser", tags: {}}))
+    .then(function(response){console.log(response.data)});
+
+    this.RefreshSite();
```

Comment:
this fires before the above request has completed, call this after the `post`, like `.then(this.RefreshSite)`. or even better, have backend to return the new post and prepend to current posts in state to avoid unnecessary fetching. e.g. `.then((res) => this.setState({ posts: [res.data, ...this.state.posts] }))`